api/v1/helpers.go

```diff
@@ -396,6 +396,11 @@ func (v *VerticaDB) GetRestartTimeout() int {
 	return vmeta.GetRestartTimeout(v.Annotations)
 }
 
+// GetCreateDBNodeStartTimeout returns the timeout value for createdb node startup
+func (v *VerticaDB) GetCreateDBNodeStartTimeout() int {
```

Comment:
What about negative values?

Reply:
I think a negative value would be okay. It should cause vclusterops to not wait for the polling. Although I have some doubts that vclusterops actually honours the timeout specified. I added some comments in the jira so that we investigate that part. But when/if it does use the timeout it will treat any negative number like its no timeout. There isn't any other way to turn off the polling.

Reply:
As discussed, we'll add a webhook check to reject negative values.